desktop/renderer-app/src/tasks/init-register-apps.ts

```diff
@@ -29,6 +29,10 @@ const registerApps = (): void => {
             const app = await import("@netless/app-geogebra");
             return app.default ?? app;
         },
+        appOptions: {
+            HTML5Codebase:
+                "https://flat-storage-cn-hz.whiteboard.agora.io/GeoGebra/HTML5/5.0/web3d",
```

Comment:
add todo comment

Reply:
We shouldn't have cn-hz in the domain name, which we should fix at a later stage